src/web/middleware/sentry.rs

```diff
@@ -95,8 +94,6 @@ where
                             tags.tags.insert(k, v);
                         }
                     };
```

Comment:
let's continue sending it to Sentry but in its extras for now

Reply:
Sadly, the lib we're using doesn't understand how to send "extras", only "tags".

Reply:
Isn't this it here? https://docs.rs/sentry/0.17.0/sentry/protocol/struct.Event.html#structfield.extra

Reply:
Ah, I might be confusing this with the cadence library. Sorry. That kind of morning.